src/erlang_ls_tree.erl

```diff
@@ -101,7 +101,7 @@ points_of_interest(Tree, Extra) ->
   try points_of_interest(Tree, Type, Extra)
   catch
     Class:Reason:Stacktrace ->
-      lager:warning( "Could not analyze tree: ~p:~p ~P"
+      lager:warning( "Could not analyze tree: ~p:~p ~p"
```

Comment:
`~P` seems to be an invalid format argument.

Reply:
The depth argument was missing.